Control/lib/config/publicConfigProvider.js

```diff
@@ -68,8 +68,8 @@ function getConsulConfig(config) {
  * @return {JSON}
  */
 function _getGrafanaConfig(config) {
-  if (config?.grafana && config?.http?.hostname && config?.grafana?.port) {
-    const hostPort = `http://${config.http.hostname}:${config.grafana.port}`;
+  if (config?.grafana && config?.grafana?.url) {
```

Comment:
Here there is no need for the first check anymore. There was no need in my previous check either 😄 
`if (config?.grafana?.url)` will do the trick